.gitignore

```diff
@@ -99,6 +99,7 @@ test/helm-operator/helm-operator
 images/scorecard-proxy/scorecard-proxy
 images/scorecard-test/scorecard-test
 images/scorecard-test-kuttl/scorecard-test-kuttl
+/hack/bin
```

Comment:
wdyt about we use the bin dir that already exist for we are able to keep all centralized instead?

Reply:
that was my original intention... but there isn't a `bin` in this project.   And I didn't see a target or script that builds to that folder... did I miss something?   Happy to transition to that location.

Reply:
We should not have the bin since it is in the gitignore. It is not a good practice to commit them. 
IMO it is better we still using for the bin/ dir in the root of the project for we do not end up have many bins directories.

Reply:
Built binaries are built to `build/`, currently nothing is built to `bin/`. I'm fine with a top-level `bin/` directory, and adding it to the `.gitignore` so binaries are not committed.

Reply:
I'm not sure where the confuse is coming from... but I would never commit this binary... not sure why that was assumed.   It was suggested in this thread that we build to `bin`.  my comment was that there currently isn't a `bin` for this project but there is an entry in the `.gitignore` file to ignore it.  which we would want to do in this case.  The question was should I start using a `bin/`... looks like we have 2 votes for that as the solution.   1 sec

Reply:
what I was originally trying to clarify is if I may have missed `bin` being used... because I didn't see it.   I questioned it, because there was a reference to `bin` in the ignore file... but I couldn't see a reason for it.

Reply:
Hi @kensipe, 

the suggestion here is to install the `golangci-lint` in the `bin/` instead of `/hack/bin`. The reason is: If we start to add bin directories in the subdirectories we can end up in a scenario where we have more than one `bin` directory which reduces the maintenance ability.

Reply:
@camilamacedo86 that is what was in the last push from several hours ago